README.md

```diff
@@ -0,0 +1,3 @@
+# on-demand-github-pages
+The course repository for GitHub 102 and 103
+bdfworks trying to follow tutorial to best of ability
```

Comment:
Make sure you're following the instructions in the course. You need to introduce a file inside the `_pages` folder with the template provided in the tutorial.

Reply:
May have become crossed up amongst tutorials...